.travis.yml

```
@@ -46,6 +46,11 @@ matrix:
       env:
         - SPARK_VERSION="master"
         - JAVA_VERSION=oraclejdk8
+    - name: "Spark master (R release, openjdk11)"
+      r: release
+      env:
+        - SPARK_VERSION="master"
+        - JAVA_VERSION=openjdk11
```

```suggestion
        - JAVA_URL=https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.1%2B13/OpenJDK11U-jdk_x64_linux_hotspot_11.0.1_13.tar.gz
```

We would also need to add after `before_install`:

```
- wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
- if [[ ! -z "$JAVA_URL" ]]; then source install-jdk.sh --url $JAVA_URL ; fi
```

See https://github.com/danielnorberg/rut/blob/master/.travis.yml